app/api/mealsController/users.controller.js

```diff
@@ -0,0 +1,73 @@
+import Joi from "joi";
+import usersDb from "../db/users.db";
+
+class UsersController {
+  // add new user
+  addNewUser(req, res) {
+    const schema = {
+      firstName: Joi.string().required(),
+      lastName: Joi.string().required(),
+      userName: Joi.string().required(),
+      password: Joi.string().required(),
+      isAdmin: Joi.boolean()
+    };
+
+    const user = Joi.validate(req.body, schema);
+
+    if (user.error) {
+      return res.status(404).send(user.error.message);
+    }
+
+    const userToAdd = {
+      id: usersDb.length + 1,
```

Comment:
Again, refrain from using the length of the array for ID. Take a look at [shortId](https://www.npmjs.com/package/shortid)